nephelaiio/ansible-collection-patroni
# Add Barman configuration playbook

- Add barman host to test scenarios
- Update README
- Rename haproxy master backend
- Add barman test scenario
- Add Barman ssh management tasks
- Fix barman verification tasks
- Parameterize all port definitions
- Add missing ssh verification port
- Add ssh daemon install to molecule prepare play
- Disable host key checks during verification steps
- Add missing vardef
- Fix barman ssh key owner and group
- Fix broken haproxy port configuration
- Drop barman server from install scenario
- Add postgresql client install tasks to barman play
- Add barman replication slot
- Refactor postgresql client install tasks
- Fix yaml formatting for patroni config file
- Add replication slot verification task
- Generalize replication slot verify
- Fix barman check errors
- Add barman backup init tasks
- Add Barman initialization wait step
- Refactor molecule tasks
- Fix lint warnings
- Fix barman validation errors
- Initialize barman crash test tasks
- Finalize barman crash test tasks
- Bump version number
- Refactor barman play to support redhat derivatives
- Add cronjob setup tasks
- Fix backup verification task
- Add barman backup verification tasks
- Refactor cluster crash test logic
- Fix verification task order
